qaframework-bdd-tests/src/test/resources/features/refapp-2.x/10-allergy/allergies.feature

```diff
@@ -1,17 +1,31 @@
 Feature: Allergies Management
 
   Background:
-    Given a user clicks on Allergies link from Patient dashboard
-    Then the system loads Allergies page
+    Given a user clicks on Allergies link from the Patient dashboard
+    Then the system loads the Allergies board page
 
   @selenium
-  @dashboard
-  Scenario: Add No known allergies
-    And a user clicks No Known Allergy button
-    Then the system add no known allergies into the allergies table
-
-  @selenium
-  @dashboard
-  Scenario: Remove No known allergies
-    And a user clicks Remove No Known Allergy icon
+  @allergies
```

Comment:
what do you think of `@allergies` vs `@allergy`?

Reply:
@kdaud since the link  is allergies on the patient dashboard, I felt it would be better to match it. What do you think?

Reply:
do we name scenario tags based on links?

Reply:
@kdaud  i think we can leave it as allergies since practically we dont deal with one allergy.